`'padded-blocks': ['error', 'never']` doesn't appear to disallow padding in ES6 `class` declarations

**Tell us about your environment**
NodeJS (literally just `index.js`, `package.json`, and a `node_modules` folder).

* **ESLint Version:** v3.12.2
* **Node Version:** v6.9.2
* **npm Version:** 3.10.9

**What parser (default, Babel-ESLint, etc.) are you using?**: Default

**Please show your full configuration:**
```js
'use strict';

const ESLintConfiguration = {
	root: true,
	parserOptions: {
		sourceType: 'script'
	},
	env: {
		es6: true,
		node: true
	},
	rules: {
		'padded-blocks': ['error', 'never']
	}
}

module.exports = ESLintConfiguration;
```

**What did you do? Please include the actual source code causing the issue.**
Linted this file:
```js
'use strict';

class SomeClass {

	constructor(opts) {

		this.opts = opts;

	}

}

const instance = new SomeClass(12);

console.log(instance.opts);
```
Specifically, I ran this command:
```
`npm bin`/eslint index.js
```

**What did you expect to happen?**
```
/Users/treybrisbane/src/personal/eslint-test/index.js
  3:17  error  Block must not be padded by blank lines  padded-blocks
  5:20  error  Block must not be padded by blank lines  padded-blocks
  9:2   error  Block must not be padded by blank lines  padded-blocks
  11:1  error  Block must not be padded by blank lines  padded-blocks

✖ 4 problems (4 errors, 0 warnings)
```

**What actually happened? Please include the actual, raw output from ESLint.**
```
/Users/treybrisbane/src/personal/eslint-test/index.js
  5:20  error  Block must not be padded by blank lines  padded-blocks
  9:2   error  Block must not be padded by blank lines  padded-blocks

✖ 2 problems (2 errors, 0 warnings)
```

Interestingly, if I use this configuration (note the more specific `padded-blocks` options)...
```js
'use strict';

const ESLintConfiguration = {
	root: true,
	parserOptions: {
		sourceType: 'script'
	},
	env: {
		es6: true,
		node: true
	},
	rules: {
		'padded-blocks': ['error', { blocks: 'never', classes: 'never', switches: 'never' }]
	}
}

module.exports = ESLintConfiguration;
```
... I get the expected output.

It seems as though simply supplying `'never'` to `padded-blocks` does NOT set `classes: 'never'`, but DOES set `blocks: 'never'` (what about `switches`?).
This may be a documentation oversight, but if this is intended behaviour, it seems a bit unintuitive!

--- a/docs/rules/padded-blocks.md
+++ b/docs/rules/padded-blocks.md
@@ -27,7 +27,7 @@ This rule has one option, which can be a string option or an object option.
 String option:
 
 * `"always"` (default) requires empty lines at the beginning and ending of block statements (except `switch` statements and classes)
-* `"never"` disallows empty lines at the beginning and ending of block statements
+* `"never"` disallows empty lines at the beginning and ending of block statements (except `switch` statements and classes)
 
 Object option:
 

